decode/utils/reference_files/reference.yaml

```diff
@@ -81,6 +81,7 @@ InOut:
   calibration_file:  # spline calib
   experiment_out:  # main output dir
   model_init:
+  checkpoint:
```

Comment:
Maybe we should make clear that this is not the output path of the ckpt but rather where we warmstart from